miqcli/collections/vms.py

```diff
@@ -37,10 +38,12 @@ class Collections(CollectionsMixin):
                   help='vendor of an vm(s)')
     @click.option('--vtype', type=str, default='',
                   help='type of an vm(s) - ex. "Openstack", "Amazon"...')
+    @click.option('-v', '--verbose', count=True,
```

Comment:
Same suggestion as my comment above.

Reply:
same comment as Ryan, instead of introducing verbose, you can just add log.debug messages and they will get displayed if verbose is set from the top level.